Documentation/ceph-upgrade.md

```diff
@@ -520,18 +523,85 @@ located.
         value: "quay.io/k8scsi/csi-snapshotter:v1.2.2"
     - name: ROOK_CSI_ATTACHER_IMAGE
         value: "quay.io/k8scsi/csi-attacher:v1.2.0"
+    - name: ROOK_CSI_RESIZER_IMAGE
+        value: "quay.io/k8scsi/csi-resizer:v0.4.0"
 ```
 
-### Verifying updates
+#### Verifying updates
 You can use the below command to see the CSI images currently being used in the cluster. Once there
 is only a single version of each image and the version is the latest one you expect, the CSI pods
 are updated.
 
 ```console
 # kubectl --namespace rook-ceph get pod -o jsonpath='{range .items[*]}{range .spec.containers[*]}{.image}{"\n"}' -l 'app in (csi-rbdplugin,csi-rbdplugin-provisioner,csi-cephfsplugin,csi-cephfsplugin-provisioner)' | sort | uniq
-quay.io/cephcsi/cephcsi:v1.2.2
+quay.io/cephcsi/cephcsi:v2.0.1
 quay.io/k8scsi/csi-attacher:v1.2.0
 quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
 quay.io/k8scsi/csi-provisioner:v1.4.0
 quay.io/k8scsi/csi-snapshotter:v1.2.2
+quay.io/k8scsi/csi-resizer:v0.4.0
+```
+### Updating cephcsi from v2.0.0 to v2.0.1
+
+If you have a v2.0 cluster with the following environment (`env`) variables:
+
+```yaml
+  env:
```

Comment:
This is repeated from lines 517-527. Can we remove it? They should know if they had the CSI driver v2.0.0 installed since it wasn't the default in any release.